src/opening-period/time-span/TimeSpan.scss

```diff
@@ -37,3 +38,13 @@
   justify-content: space-between;
   align-items: flex-end;
 }
+
+.checkbox-container {
+  display: flex;
+  align-items: center;
+  height: var(--spacing-3-xl);
+}
+
+.time-span-end-time-input-wrapper {
+  margin-right: 140px;
```

Comment:
Could this be replaced with some predefined design token? So there would be some rhythm in the layout: '$spacing-layout-2-xl'

Reply:
- [ ] Apply design token here instead of hard coded value